lib/ansible/parsing/vault/__init__.py

```diff
@@ -147,20 +147,11 @@ def is_encrypted_file(file_obj, start_pos=0, count=-1):
     current_position = file_obj.tell()
     try:
         file_obj.seek(start_pos)
-        vaulttext = file_obj.read(count)
```

Comment:
I don't remember why the options to the inner to_text()  here different or if it is significant.

``` python
vaulttext, encoding='ascii', errors='strict'```

vs

``` python
data, encoding='ascii', errors='strict', nonstring='strict'
```